client/app/deskaas/deskaas-details/deskaas-details.html

```diff
@@ -1,165 +1,228 @@
-                                <i class="oui-icon oui-icon-chevron-right" aria-hidden="true"></i>
-                            </button>
-                        </li>
-                        <li class="cui-tile__item cui-tile__item_button cui-tile__top-bordered">
-                            <button class="oui-button oui-button_link oui-button_icon-right oui-button_full-width cui-tile__button"
-                                type="button"
-                                data-ng-click="DeskaasDetailsCtrl.actions.remove.callback()"
-                                data-ng-disabled="!DeskaasDetailsCtrl.actions.remove.isAvailable()">
-                                <span data-ng-bind="DeskaasDetailsCtrl.actions.remove.text"></span>
-                                <i class="oui-icon oui-icon-chevron-right" aria-hidden="true"></i>
-                            </button>
-                        </li>
-                    </ul>
-                </div>
+                <oui-tile data-heading="{{::'common_tile_shortcut' | translate}}" class="h-100 oui-tile-h-100">
+                    <oui-tile-button
+                        data-text="{{::$ctrl.actions.restart.text}}"
+                        data-disabled="!$ctrl.actions.restart.isAvailable()" 
+                        data-on-click="$ctrl.actions.restart.callback()">
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-text="{{::$ctrl.actions.changePassword.text}}"
+                        data-disabled="!$ctrl.actions.changePassword.isAvailable()" 
+                        data-on-click="$ctrl.actions.changePassword.callback()">
+                    </oui-tile-button>
+                    <oui-tile-button
+                        data-text="{{::$ctrl.actions.accessConsole.text}}"
+                        data-disabled="!$ctrl.actions.accessConsole.isAvailable()" 
+                        data-on-click="$ctrl.actions.accessConsole.callback()">
+                    </oui-tile-button>
+                    <oui-tile-button 
+                        data-text="{{::$ctrl.actions.reinit.text}}"
+                        data-disabled="!$ctrl.actions.reinit.isAvailable()"
+                        data-on-click="$ctrl.actions.reinit.callback()">
+                    </oui-tile-button>
+                    <oui-tile-button 
+                        data-text="{{::$ctrl.actions.changeOffer.text}}" 
+                        data-disabled="!$ctrl.actions.changeOffer.isAvailable()"
+                        data-on-click="$ctrl.actions.changeOffer.callback()">
+                    </oui-tile-button>
+                    <oui-tile-button 
+                        data-text="{{::$ctrl.actions.remove.text}}" 
+                        data-disabled="!$ctrl.actions.remove.isAvailable()"
+                        data-on-click="$ctrl.actions.remove.callback()">
+                    </oui-tile-button>
```

Comment:
Maybe you could use a `data-ng-repeat` on `$ctrl.actions` if all actions are displayed here